elf/elf.go

```diff
@@ -460,6 +460,7 @@ func (b *Module) Load(parameters map[string]SectionParams) error {
 	if err != nil {
 		return err
 	}
+	defer b.file.Close()
```

Comment:
https://golang.org/pkg/debug/elf/#File.Close

> If the File was created using NewFile directly instead of Open, Close has no effect.

The file was created with `elf.NewFile`, so it does not seem the correct way to close it.